cob-pcs/cob.bash.bashrc.t

```diff
@@ -1,2 +1,6 @@
+function parse_git_branch () {
+  git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ (\1)/'
+}
+
 # set color fo all users
-PS1='${debian_chroot:+($debian_chroot)}\[\033[01;34m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+PS1='${debian_chroot:+($debian_chroot)}\[\033[01;34m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[0;33m\]$(parse_git_branch)\[\033[0m\]\$ '
```

Comment:
The color:

base: ${debian_chroot:+($debian_chroot)}\[\033[01;**31**m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[0;33m\]$(parse_git_branch)\[\033[0m\]\$ '
torso: '${debian_chroot:+($debian_chroot)}\[\033[01;**34**m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[0;33m\]$(parse_git_branch)\[\033[0m\]\$ '
head: '${debian_chroot:+($debian_chroot)}\[\033[01;**33**m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[0;33m\]$(parse_git_branch)\[\033[0m\]\$ '